### ISSUE TITLE
cdn.bootcss.com is blocked as a CDN

### ISSUE THREAD

**Issue by l5h5t7** [EFForg/privacybadger#1358]
Sample sites: 
- https://myssl.com/
- https://www.runoob.com/

**Comment by ghostwords** [comment]
I'm trying to reproduce but my Badger doesn't see any tracking from this CDN. I have a number of reports of mysterious false positives (?) at this point, will label them all.

Could you share your snitch map for `cdn.bootcss.com`? See [here](https://github.com/EFForg/privacybadger/issues/1338#issuecomment-296402505) for instructions (just replace "metapix" with "bootcss").

Can you reproduce if you install Privacy Badger into a new profile and visit these sites? Does Badger say that `cdn.bootcss.com` "doesn't appear to be tracking you", or is the domain listed as a potential tracker?

**Comment by l5h5t7** [comment]
My snitch map:
```
VM97:3 **** ACTION_MAP for bootcss
VM97:5 bootcss.com {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1492913577700,
  "userAction": ""
}
VM97:5 cdn.bootcss.com {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1493722364480,
  "userAction": "user_cookieblock"
}
VM97:7 **** SNITCH_MAP for bootcss
VM97:9 bootcss.com [
  "aliyun.com",
  "sysgeek.cn",
  "runoob.com"
]
```
Does Badger says "doesn't appear to be tracking you" into a new profile when first visit these sample sites.

**Comment by ghostwords** [comment]
This bug should be fixed in Privacy Badger version 2017.6.13.1 (by #1403 and #1428). Please let me know if you run into it again with the new version.
